tests/e2e/pageObjects/components/browser/bulk-actions.ts

```diff
@@ -41,6 +41,8 @@ export class BulkActions {
      * Open Bulk Actions and confirm deletion
      */
     async startBulkDelete(): Promise<void> {
+        // Wait for Bulk actions animation ends
+        await t.wait(1000);
```

Comment:
can you do something like this 
await t.expect(this.actionButton.exists).ok('', { timeout: 10000 });

Reply:
no, it won't work for testCafe
click(this.actionButton) already checks this, and it is visible on animation start
and testCafe is trying to click this button when animation not ended and it leads to misclick

Reply:
it is very sad